demo/data.js

```diff
@@ -2087,11 +2087,11 @@ export const demoData = {
       right: { style: "dashed", color: "#008800" },
       bottom: { style: "dotted", color: "#0000FF" },
     },
-    7: { bottom: ["thin", "#000"] },
-    8: { bottom: ["thin", "#000"], left: ["thin", "#000"] },
-    9: { bottom: ["thin", "#000"], right: ["thin", "#000"] },
-    10: { top: ["thin", "#000"], left: ["thin", "#000"] },
-    11: { top: ["thin", "#000"], right: ["thin", "#000"] },
+    11: { bottom: ["thin", "#000"] },
```

Comment:
Shouldn't it be something like `11: { bottom: { style: "thin", color": "#000"}}` ? Same comments for the lines below

Reply:
Indeed. Some borders are also not needed anymore since the introduction of table style in demo data :)